mir-ci/mir_ci/fixtures/servers.py

```diff
@@ -157,7 +158,7 @@ def miriway(*args, channel="stable", classic=True, **kwargs):
     return snap("miriway", *args, channel=channel, classic=classic, **kwargs)
 
 
-@server(ServerCap.ALL ^ (ServerCap.DISPLAY_CONFIG | ServerCap.SCREENCOPY | ServerCap.INPUT_METHOD))
+@server(ServerCap.ALL ^ (ServerCap.DISPLAY_CONFIG | ServerCap.SCREENCOPY | ServerCap.INPUT_METHOD | ServerCap.VNC))
```

Comment:
Does gnome shell really not have VNC support?

Reply:
I'll change this. It should support VNC. I think I'm just out of luck trying to make it work locally. The shell crashes every time I enable VNC.